test/tests/functional/pbs_PP483_PP484_PP485.py

```diff
@@ -0,0 +1,103 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2016 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# The PBS Pro software is licensed under the terms of the GNU Affero General
+# Public License agreement ("AGPL"), except where a separate commercial license
+# agreement for PBS Pro version 14 or later has been executed in writing with Altair.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and distribute
+# them - whether embedded or bundled with other software - under a commercial
+# license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+import json
+
+
+class TestPP483_PP484_PP485(TestFunctional):
+    """
+    validate qstat output in dsv,json and oneline format
+    """
+
+    def test_PP483_PP485(self):
+        """
+        submit a single job and check the no of attributes parsed from dsv is equal
+        to the one parsed from one line output.
+        """
+        j = Job(TEST_USER)
+        j.set_sleep_time(10)
+        jid = self.server.submit(j)
+        time.sleep(1)
+        qstat_cmd = os.path.join(
+            self.server.pbs_conf['PBS_EXEC'], 'bin', 'qstat')
+        f = open('qstat_dsv_script', 'w')
+        f.write(qstat_cmd + ' -f -F dsv ' + str(jid) + ' > qstat_dsv.out')
+        f.close()
```

Comment:
this will not work if this test is run in a directory where the user does not have permission to create files...